frontend/src/features/admin/users/__snapshots__/ManageUsers.test.tsx.snap

```diff
@@ -45,7 +45,7 @@ exports[`Manage Users Component Snapshot matches 1`] = `
         </button></div>
     </div>
   </form>
-  <div class=\\"sc-fzqBkg iqmWnC container-fluid\\">
+  <div class=\\"sc-fzoydu kEdtJm container-fluid\\">
```

Comment:
I think you fixed this, but it does seem to be broken again, one of us will have to take another look at some point.

Reply:
maybe was only fixed for enzyme snapshots?